### ISSUE TITLE
Highlight style for "Visual" has very low contrast

### ISSUE THREAD

**Issue by nick4tech** [AlphaTechnolog/everblush.nvim#9]
The highlight style often used to identify all instances of a variable on a document ("Visual"), makes the name unreadable because it doesn't have enough contrast with the color for  variable names.

This variable name:
![image](https://user-images.githubusercontent.com/6836442/184828151-278d8897-645d-4138-a855-a88f93917377.png)

Looks like this when cursor is hovering over them:
![image](https://user-images.githubusercontent.com/6836442/184828269-ad59d43b-7849-431e-90a1-3ecbb11ce6dc.png)

**Comment by mngshm** [comment]
Do you mean when switching to visual mode its not readable ?

**Comment by nick4tech** [comment]
No, because when highlighting text in visual mode, it's actually readable:

![image](https://user-images.githubusercontent.com/6836442/184839264-37d1c8cd-4b02-4f25-9596-9bc482138b33.png)

**Comment by mngshm** [comment]
ohh so what's your issue ? Sorry, I am not able to understand what you mean

**Comment by nick4tech** [comment]
I'm using [this](https://github.com/RRethy/vim-illuminate) plugin to highlight all instances of a variable in the code. So for example with the colorscheme OceanicNext it looks like this:
![image](https://user-images.githubusercontent.com/6836442/184853459-003e18e2-4e34-4862-87ac-fb58d0f2ca1c.png)

However, in everblush the color used for this highlight doesn't provide good contrast with the variable color, so it actually looks like this:
![image](https://user-images.githubusercontent.com/6836442/184853620-36a8a084-8762-4559-809b-a64b8c1ab92b.png)

Which makes the variables unreadable.

Hope that clears it up. Let me know if you need anything else.

**Comment by mngshm** [comment]
OK now I got your issue. I have to add support for this Plugin. I'll update when it's done
